xc7/fasm2bels/README.md

```diff
@@ -0,0 +1,60 @@
+FASM 2 BELs
+-----------
+
+fasm2bels is a tool designed to convert FASM generated from a bitstream back
```

Comment:
fasm2bels is a tool designed to take a FASM file into Vivado.

It does this by generating a file describing the BEL connections (techmapped Verilog) and TCL commands for Vivado which lock the BEL placements.

This makes it possible to perform simulation and analysis of a FASM file inside Vivado.

In the absence of bugs, it is expected that after consuming the BEL connections and TCL constraints Vivado will produce a bitstream identical to the bitstream the FASM file would generate.